### PR TITLE
Added body tracking to solver handler

### PR DESCRIPTION
**Pull request by StephenHodgson**
# XRTK - Mixed Reality Toolkit Change Request

## Overview

Added the ability to specify the body (the mixed reality playspace) as a tracked object to solve the position and rotation against. Used for body locked solvers.

## Breaking Changes:
- Renamed `TrackedObjectType.MotionControllerLeft` to `TrackedObjectType.LeftHandOrController`
- Renamed `TrackedObjectType.MotionControllerRight` to `TrackedObjectType.RightHandOrController`

### CODE REVIEW

File: XRTK-Core/Packages/com.xrtk.core/Definitions/Utilities/TrackedObjectType.cs [hunk 1]
```diff
@@ -16,6 +16,10 @@ public enum TrackedObjectType
         /// <summary>
         /// Calculates position and orientation from the right motion-tracked controller.
```
Comment (SimonDarksideJ) [inline XRTK-Core/Packages/com.xrtk.core/Definitions/Utilities/TrackedObjectType.cs]: Right controller, not just mr motion controller

Comment (StephenHodgson) [reply XRTK-Core/Packages/com.xrtk.core/Definitions/Utilities/TrackedObjectType.cs]: That's a breaking change but sure

**Comment by StephenHodgson** [comment]
I'll be sure to do that with the next release

**Comment by StephenHodgson** [comment]
So I realized I might have made a mistake here. I didn't actually setup a `BodyLock` but a `PlayspaceLock`

When the user walks around their space the tracked objects don't update correctly with the head/body position. We'll probably need to update the rig appropriately.

**Pull request merged**